### PR TITLE
feat(WebexComponents): add pressed state on "Participant", "Share" and "Collapse" buttons

### PR DESCRIPTION
**Pull request by patricia0817**
Add active state on "Participant", "Share" and "Collapse" buttons
Task link: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-267952

### CODE REVIEW

File: src/adapters/MeetingsJSONAdapter/controls/RosterControl.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ export default class RosterControl extends MeetingControl {
       icon: 'participant-list_28',
       text: 'Participants',
       tooltip: 'Hide participants panel',
+      pressed: true,
     };
```
Comment (alinasuciu) [inline src/adapters/MeetingsJSONAdapter/controls/RosterControl.js:38]: Please try to add an extra class if button has different style in order to simplify the logic.

**Review by alinasuciu — APPROVED** [review]

File: src/components/WebexMeetingControl/WebexMeetingControl.jsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ import {useMeetingControl} from '../hooks';
 import Button from '../generic/Button/Button';
 import Icon from '../generic/Icon/Icon';
 import Select from '../generic/Select/Select';
+import {ROSTER_CONTROL, SHARE_CONTROL} from '../../adapters/MeetingsJSONAdapter';
```
Comment (lalli-flores) [inline src/components/WebexMeetingControl/WebexMeetingControl.jsx:10]: You should not import from adapters into the component code. How would work for other adapters? e.g. SDK adapter?

File: src/components/WebexMeetingControl/WebexMeetingControl.jsx [hunk 3]
```diff
@@ -28,11 +29,15 @@ const controlTypeToButtonType = {
...
   const {
     icon,
+    ID,
     type,
     text,
     tooltip,
   } = display;
   const isDisabled = display.state === MeetingControlState.DISABLED;
+  const isActive = display.state === MeetingControlState.ACTIVE;
+  const isPressed = (ID === ROSTER_CONTROL || ID === SHARE_CONTROL) && isActive;
```
Comment (lalli-flores) [inline src/components/WebexMeetingControl/WebexMeetingControl.jsx:39]: Why are you hard coding specifically for this buttons? how would this work if we add another button that should behave this way? This is not scalable

**Review by lalli-flores — CHANGES_REQUESTED** [review]

File: src/components/WebexMeetingControl/WebexMeetingControl.jsx [hunk 4]
```diff
@@ -47,7 +50,7 @@ function renderButton(action, display, style, showText, asItem) {
   } else {
     output = (
       <Button
-        className="control-button"
+        className={`control-button ${isPressed ? 'active' : ''}`}
         type={controlTypeToButtonType[type] || 'default'}
```
Comment (alinasuciu) [inline src/components/WebexMeetingControl/WebexMeetingControl.jsx:54]: I think it's enough to check if is active and add the class active although all the buttons will take this class if they are active and in css just add style for active class on pressable buttons.

File: src/components/WebexMeetingControl/__snapshots__/WebexMeetingControl.stories.storyshot [hunk 5]
```diff
@@ -6,7 +6,7 @@ Array [
     className="wxc-meeting-control"
   >
     <button
-      className="wxc-button control-button wxc-button--default"
+      className="wxc-button control-button  wxc-button--default"
       disabled={false}
```
Comment (alinasuciu) [inline src/components/WebexMeetingControl/__snapshots__/WebexMeetingControl.stories.storyshot:10]: correct this white space

**Pull request closed without merging**